docs/topics/rn-new-features.adoc

```diff
@@ -7,66 +7,22 @@
 
-The current release contains the {ProductName} Community Operator for installing the {WebName} on {ocp-short} 4.6 and later versions.
+.Rules for migrating from JBoss EAP 7.3 to Boss EAP 7.4
 
-.Installation in multiple namespaces on {ocp-short}
+The current release contains rules for migrating from JBoss EAP 7.3 t o Boss EAP 7.4. The target of these rules is `eap:7`.
 
-The current release can be installed in multiple namespaces on {ocp-short}. Previously, the {WebName} could be installed in only one namespace.
+.Rules for migrating from JBoss EAP XP 2 to JBoss EAP XP 3
 
-.Additional rules for migrating from Spring Boot to Quarkus
-
-The current release contains additional rules for migrating from Spring Boot to Quarkus. The target of these rules is `quarkus`.
-
-The new rules identify Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. The rules also report Spring artifacts that do not have Quarkus equivalents.
-
-The rules identify application dependencies for which there are equivalent Quarkus extensions, including the following:
-
-* Agroal
-* Amazon DynamoDB
-* Amazon IAM
-* Amazon S3
-* Amazon Simple Email Service
-* Amazon Simple Notification Service
-* Amazon Simple Queue Service
-* AWS Config
-* AWS Key Management Service
-* AWS Lambda with HTTP APIs
-* AWS Lambda with AWS X-Ray
-* Apache Avro
-* Apache Kafka
-* Artemis
-* Hibernate
-* Java Database Connectivity rules for the following databases:
-** Apache Derby
-** DB2
-** MariaDB
-** MySQL
-** PostgreSQL
-* JSONB
-* JSONP
-* Keycloak
-* Kubernetes
-* Narayana
-* OpenShift Container Platform
-* RESTeasy
-* WildFly Elytron
-
-.Rules for migrating to the latest version of Quarkus
-
-The current release contains rules for migrating to the latest version of Quarkus. The target of these rules is `quarkus<version>`, for example, `quarkus1:13` for Quarkus 1.13.x.
-
-These rules ensure that the applications are migrated to a target that has the latest features, resolved issues, and patches.
-
-These rules will be updated for future Quarkus releases.
+The current release contains rules for migrating from JBoss EAP XP 2 to JBoss EAP XP 3 The target of these rules is `eapxp:3`.
 
 .Rules for migrating from Thorntail to JBoss EAP XP 2
 
 The current release contains rules for migrating from Thorntail to JBoss EAP XP 2. The target of these rules is `eapxp:2`.
 
-For more information about this migration path, see the link:https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html/red_hat_jboss_eap_xp_2.0.0_migration_guide/thorntail-application-maven-project-migration_default#migrating-a-thorntail-application-maven-project-into-eap-xp_default[Red Hat JBoss EAP XP 2.0.0 Migration Guide].
+These rules were contained in {ProductShortName} 5.1.4; notice of their inclusion is repeated for customers who may not have installed that version.
```

Comment:
Huh? What are you trying to say? I am not sure you need the 2nd clause. Release notes are for people who are installing a version, not for those who have not installed it. Is there any reason to include this sentence? Rather than phrasing it like a "new feature", if you really need to mention it, you can make it a note like "Rules for migrating from Thorntail to JBoss EAP XP 2 have been included since version 5.1.4." Putting it into "new features" when it is not a new feature looks a bit weird.